src/components/game/Game.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component}  from 'react';
+import {Board} from '../board/Board';
+import * as boardHelper from '../board/boardHelper';
+import * as gameHelper from '../game/gameHelper';
+import * as lineAnalysis from '../../lineAnalysis';
+import * as cellValue from '../../cellValue';
+
+export class Game extends Component {
+  constructor(props){
+    super(props);
+    this.state={
+      grid: Array(this.props.boardSize).fill(cellValue.EMPTY)
+    };
+    this.handleClick=this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    const index=e.target.id;
+    const grid=this.state.grid.slice();
+    if (boardHelper.isMoveAvailable(grid, index) && !lineAnalysis.isGameOver(grid)) {
+      grid[index]=gameHelper.determineMark(this.state.grid)
+    }
+    this.setState({grid: grid});
+  }
+
+  render() {
+    return (
+      <div>
+      <Board updatedGrid={this.state.grid}
+        onClick={this.handleClick}/>
```

Comment:
Might the naming of the board's `onClick` prop interfere with React's [onClick](https://reactjs.org/docs/handling-events.html). 

If I'm reading this correctly, I feel like the only `onClick` is in the Cell's button. Might be better to give the rest of them other names? 

Though I could be wrong on this!